compiler-plugin-tests/build.gradle

```diff
@@ -103,9 +103,9 @@ def tomlVersion = stripBallerinaExtensionVersion("${project.version}")
 task updateTomlFiles {
     doLast {
         def updateTomlFile = { File file ->
-            def ballerinaTomlFilePlaceHolder = new File("${project.rootDir}/build-config/resources/BallerinaTest.toml")
+            def ballerinaTomlFilePlaceHolder = new File("${project.rootDir}/build-config/resources/CompilerPluginTest.toml")
```

Comment:
Is this change correct? There's still a BallerinaTest.toml file?

Reply:
Its for `ballerina-test`. This is for `compiler-plugin` tests.
I will cehck

Reply:
What was the conclusion on this?